requirements.txt

```diff
@@ -1,9 +1,12 @@
 # pythonwhat deps
-protowhat~=2.1.0
+protowhat~=2.1.3
 asttokens~=2.0.5
 dill~=0.3.4
-markdown2~=2.3.10
+markdown2~=2.4.6
 jinja2~=2.11.3
+markupsafe==2.0.1
```

Comment:
Can adding these dependencies be avoided? Or are all 3 absolutely necessary?

Reply:
markupsafe was added because jinja2 uses it as a dependency and it didn't fix the version. Older versions break the package because there's a breaking change. Pygments is needed for the markdown2 extras.